### ISSUE TITLE
[Request]Support for IC2 fluid reactors

### ISSUE THREAD

**Issue by panda2134** [OpenMods/OpenPeripheral-Integration#25]
As the output of an EU reactor is throttled,the fluid reactors have come into use widely.So being able to control them by using a CC computer seems important. It would be great if the support for them is added. I'm happy to open a pull request to do this if you like this idea.

**Comment by boq** [comment]
What API call do you want to expose? We already do parts of reactor itself: https://github.com/OpenMods/OpenPeripheral-Integration/blob/master/src/main/java/openperipheral/integration/ic2/AdapterReactor.java#L23, but we may have missed something.

Though parts of reactor vessel seem to be plain blocks (so no interaction at all), while others have no public (or even internal) interface - for example, liquid port exposes only fluid handler functions.

**Comment by panda2134** [comment]
In TileEntityReactorRedstonePort.java , I found a public method called getReactor,which means that we can interact with the reactor itself by simply calling the method of the redstone port tile .

**Comment by panda2134** [comment]
btw,the method returns TileEntity,while that one in TileEntityReactorAccessHatch.java returns IInventory

**Comment by boq** [comment]
Yeah, I know. But I've asked "What API call", not "What internal method from TE". Using internal method is generally very risky and creates problems with both OpenPeripheral and integrated mod.
